compiler/passes/parseAndConvert.cpp

```diff
@@ -1380,6 +1393,8 @@ static const char* searchThePath(const char*      modName,
   const char* retval   = NULL;
 
   forv_Vec(const char*, dirName, searchPath) {
+    if (!isDirectory(dirName)) continue;
```

Comment:
This looks fine as is, but it looks like neither of those functions still live in the file that's on main today so I'll take another look after the rebase.

Reply:
Thanks for noticing! I'm going to have to make this a dyno compiler change now, as it looks like all the stuff was snipped out by one of Michael's recent PRs.